qiskit/visualization/pulse_v2/interface.py

```diff
@@ -0,0 +1,415 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name, missing-return-type-doc
+
+"""Qiskit pulse drawer.
+
+This module provides a common user interface of the pulse drawer.
+The `draw` function takes a pulse program to visualize with a stylesheet and
+a backend information along with several control arguments.
+The drawer canvas object is internally initialized from the input data and
+the configured canvas is passed to the one of plotter APIs to generate a visualization data.
+"""
+
+from typing import Union, Optional, Dict, Any, Tuple, List
+
+from qiskit.providers import BaseBackend
+from qiskit.pulse import Waveform, ParametricPulse, Schedule
+from qiskit.pulse.channels import PulseChannel
+from qiskit.visualization.exceptions import VisualizationError
+from qiskit.visualization.pulse_v2 import core, device_info, stylesheet, types
+
+
+def draw(program: Union[Waveform, ParametricPulse, Schedule],
+         style: Optional[Dict[str, Any]] = None,
+         backend: Optional[BaseBackend] = None,
+         time_range_dt: Optional[Tuple[int, int]] = None,
+         time_range_ns: Optional[Tuple[int, int]] = None,
+         disable_channels: Optional[List[PulseChannel]] = None,
+         show_snapshot: bool = True,
+         show_framechange: bool = True,
+         show_waveform_info: bool = True,
+         show_barrier: bool = True,
+         plotter: str = types.Plotter.Mpl2D,
```

Comment:
Is this more accurate?
```suggestion
         plotter: Union[str, types.Plotter] = types.Plotter.Mpl2D,
```

Reply:
or should it be
```suggestion
         plotter: str = types.Plotter.Mpl2D.value(),
```
?